cli/cmd/lql.go

```diff
@@ -257,10 +257,28 @@ func inputQueryFromURL(url string) (query string, err error) {
 func inputQueryFromEditor(action string) (query string, err error) {
 	prompt := &survey.Editor{
 		Message:  fmt.Sprintf("Type a query to %s", action),
-		FileName: "query*.lql",
+		FileName: "query*.yaml",
+	}
+
+	if action == "create" {
+		prompt.Default = `queryId: YourQueryID
+queryText: |-
+  {
+      source {
```

Comment:
one observation is the use of alias `sources` and non-alias `show-source` vs. `describe`.  Not sure if it matters that much as both get you to the same outcome...

Reply:
updated! 💯